cmd/minishift/cmd/openshift/service.go

```diff
@@ -19,39 +19,27 @@ package openshift
 import (
 	"fmt"
 	"os"
-	"strings"
-	"text/template"
 
 	"github.com/docker/machine/libmachine"
 	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 
-	"github.com/minishift/minishift/pkg/minikube/cluster"
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
+
+	"github.com/minishift/minishift/pkg/minishift/openshift"
 )
 
 var (
-	namespace          string
-	serviceURLMode     bool
-	serviceURLFormat   string
-	serviceURLTemplate *template.Template
-	https              bool
+	namespace      string
+	serviceURLMode bool
```

Comment:
@gbraad `urlMode`  might be good fit here.

See comment https://github.com/minishift/minishift/issues/342#issuecomment-286039538 .

```
$ minishift openshift service nationalparks-katacoda -n myproject --url
http://nationalparks-katacoda-myproject.192.168.99.100.nip.io/

```